src/getConfig.ts

```diff
@@ -18,7 +18,12 @@ function getConfig(): Promise<Config | null> {
         return;
       }
 
-      resolve(JSON.parse(stdout));
+      try {
+        resolve(JSON.parse(stdout));
+      } catch (e) {
+        console.error(`JSON.parse error: ${e}`);
+        reject(e);
+      }
```

Comment:
Wdyt about this?

```suggestion
      try {
        const config = JSON.parse(stdout);
        resolve(config);
      } catch (e) {
        console.error(`Failed to parse the output of "npc react-native config" to JSON. Are you sure the output returns a JSON-only string?\nError: ${e}`);
        reject(e);
      }
```

Also as a side question, do we know how the rejected case will be handled in the calling code?

Reply:
Fixed 😎

Reply:
> Also as a side question, do we know how the rejected case will be handled in the calling code?

Well, I don't know 🙈 But my assumption is that it will crash the process and print the error